scalafix/src/main/scala/fix/MimeRules.scala

```diff
@@ -0,0 +1,20 @@
+package fix
+
+import scalafix.v1._
+import scala.meta._
+
+object MimeRules {
```

Comment:
This one is great, because that was a really tedious change.

Reply:
I hope I covered most of the cases, waiting for real world feedback on this one :)

Reply:
I improved these in https://github.com/http4s/http4s/pull/2214/commits/4edd9ccadcb1735d1e1f67f6e1aa483db6eb4aa3 . Now should be able to handle more use cases (and imports as well)